src/locale/messages.fr.xlf

```diff
@@ -672,6 +672,12 @@
           <context context-type="sourcefile">src/app/modules/lti/pages/lti/lti.component.html</context>
           <context context-type="linenumber">13</context>
         </context-group>
+      </trans-unit><trans-unit id="2205921893315386228" datatype="html">
+        <source> Something wrong occurred during the login process, please <x id="START_TAG_SPAN" ctype="x-span" equiv-text="&lt;span class=&quot;alg-link&quot; (click)=&quot;restartProcess()&quot;>"/>retry<x id="CLOSE_TAG_SPAN" ctype="x-span" equiv-text="&lt;/span>"/>. <x id="LINE_BREAK" ctype="lb" equiv-text="&lt;br>"/> If the problem persists, please contact us. </source><target state="new"> Something wrong occurred during the login process, please <x id="START_TAG_SPAN" ctype="x-span" equiv-text="&lt;span class=&quot;alg-link&quot; (click)=&quot;restartProcess()&quot;>"/>retry<x id="CLOSE_TAG_SPAN" ctype="x-span" equiv-text="&lt;/span>"/>. <x id="LINE_BREAK" ctype="lb" equiv-text="&lt;br>"/> If the problem persists, please contact us. </target>
```

Comment:
the link, click, ... is in the translated file? looks suspicious

Reply:
weird, I'll regenerate translations

Reply:
But these are just other developer errors 😁. It is because you have link and html entities in your `i18n` block. 
Actually, I am not 100% sure we are not supposed to do that but that looks better to me not to have them. If you fix the link in the code, translations should be have a broken version. Read a bit about i18n to see what's the best way to proceed.

Reply:
To me it's a pretty common use case to translate text with nested tags like links for instance, and if you fix the link in the code then the translations are re-generated, only the placeholder will change. It's a normal way to interpolate translations, in the example/link I provided upper, variable interpolation works exactly the same way:
```xml
<x id="INTERPOLATION" equiv-text="{{ grantedPermission.sourceGroup.name }}"/>
```
Here if we change the variable name `grantedPermission` then as you imagine it we would have a broken version as well but it's not the case.

Although it is mentioned absolutely nowhere in the Angular documentation regarding i18n (:broken_heart:), I could find issues talking something else but giving examples of template translations with nested tags:
- https://github.com/angular/angular/issues/18302
- https://github.com/mgechev/codelyzer/issues/442
- https://phrase.com/blog/posts/localize-angular-applications-with-built-in-i18n-module/ (check out the "Working with Select Input" section)

Anyway, if we don't do it with nested tags like this …
```html
<span i18n>
  Hi there, check out this awesome <a [routerLink]="…">blog post</a> …
</span>
```
… there's no way not to split the translation parts

What do you suggest to fix the link in code ?

Reply:
Ok so `<x...` tab is done for that. Let's keep it like this for now at least.